plaidbench.py

```diff
@@ -257,30 +260,65 @@ def main():
         outputs['run_configuration'] = data.copy()
     else:
         networks.append(args.module)
+        batch_list.append(args.batch_size)
 
     for network in networks:
         printf("\nCurrent network being run : " + network)  
```

Comment:
Remove this and combine with something like:

`"Running {0} examples of {1}, batch size {2}".format(args.examples, network, batch)`